lib/src/device.dart

```diff
@@ -1,20 +1,15 @@
 import 'dart:async';
-import 'dart:ui';
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 
-import 'extensions/vector2.dart';
-
-/// Some utilities that did not fit anywhere else.
+/// Provides methods for controlling the device (e.g. setting the screen to fullscreen).
 ///
-/// To use this class, access it via [Flame.util].
-class Util {
+/// To use, access it via [Flame.device]
+class Device {
   /// Sets the app to be fullscreen (no buttons bar os notifications on top).
-  ///
-  /// Most games should probably be this way.
   Future<void> fullScreen() {
     if (kIsWeb) {
+      // TODO: We probably could use dart:html and implement this for web as well
```

Comment:
can JS change the browser window size? I wonder if this TODO still makes sense

Reply:
Make the browser fullscreen != change its window dimensions.

I have already used JS to make the browser fullscreen in plain JS, so this TODO do make sense. I plan on do a follow up to add this feature after this is merged.